src/test/java/seedu/address/logic/commands/MeetingsCommandTest.java

```diff
@@ -0,0 +1,77 @@
+package seedu.address.logic.commands;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static seedu.address.commons.core.Messages.MESSAGE_MEETINGS_LISTED_OVERVIEW;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalPersons.STEVE_MEETING;
+import static seedu.address.testutil.TypicalPersons.TYLER_MEETING;
+import static seedu.address.testutil.TypicalPersons.getScheduledAddressBook;
+
+import java.util.Arrays;
+
+import org.junit.Test;
+
+import seedu.address.logic.CommandHistory;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.meeting.Meeting;
+import seedu.address.model.meeting.SameMeetingDayPredicate;
+
+/**
+ * Contains integration tests (interaction with the Model) for {@code FindCommand}.
+ */
+public class MeetingsCommandTest {
+    private Model model = new ModelManager(getScheduledAddressBook(), new UserPrefs());
+    private Model expectedModel = new ModelManager(getScheduledAddressBook(), new UserPrefs());
+    private CommandHistory commandHistory = new CommandHistory();
+
+    @Test
+    public void equals() {
+        SameMeetingDayPredicate firstPredicate =
+                new SameMeetingDayPredicate(new Meeting("1202181800"));
+        SameMeetingDayPredicate secondPredicate =
+                new SameMeetingDayPredicate(new Meeting("1302181800"));
```

Comment:
You might want to include some tests on the different date formats as well, like `23/02/18`.

Reply:
Hey Denzel, thanks for flagging this, but since the formatting is done by the parser, a new meeting cannot be instantiated with other formats. The parser will always ensure that all formats will be changed into the DDMMYYHHMM format, which is then used to instantiate the Meeting object.